prowler/providers/aws/services/organizations/organizations_service.py

```diff
@@ -0,0 +1,233 @@
+                        self.policies.append(
+                            Policy(
+                                arn=policy["Arn"],
+                                id=policy["Id"],
+                                type=policy["Type"],
+                                aws_managed=policy["AwsManaged"],
+                                content=policy_content,
+                                targets=policy_targets,
+                            )
+                        )
+        except Exception as error:
+            logger.error(
+                f"{self.region} -- {error.__class__.__name__}[{error.__traceback__.tb_lineno}]: {error}"
+            )
+
+        finally:
+            return self.policies
+
+    def __describe_policy__(self, policy_id):
+        logger.info("Organizations - Describe policy: %s ...", policy_id)
+
+        # This operation can be called only from the organization’s management account or by a member account that is a delegated administrator for an Amazon Web Services service.
+        try:
+            policy_desc = self.client.describe_policy(PolicyId=policy_id)["Policy"]
+            policy_content = policy_desc["Content"]
+            policy_content_json = json.loads(policy_content)
+        except Exception as error:
+            logger.error(
+                f"{self.region} -- {error.__class__.__name__}[{error.__traceback__.tb_lineno}]: {error}"
+            )
+        finally:
+            return policy_content_json
+
+    def __list_targets_for_policy__(self, policy_id):
+        logger.info("Organizations - List Targets for policy: %s ...", policy_id)
+
+        try:
+            targets_for_policy = self.client.list_targets_for_policy(
+                PolicyId=policy_id
+            )["Targets"]
+        except Exception as error:
+            logger.error(
+                f"{self.region} -- {error.__class__.__name__}[{error.__traceback__.tb_lineno}]: {error}"
+            )
+        finally:
+            return targets_for_policy
+
+    def __list_delegated_administrators__(self):
+        logger.info("Organizations - List Delegated Administrators")
+
+        try:
+            list_delegated_administrators_paginator = self.client.get_paginator(
+                "list_delegated_administrators"
+            )
+            for page in list_delegated_administrators_paginator.paginate():
+                for delegated_administrator in page["DelegatedAdministrators"]:
+                    self.delegated_administrators.append(
+                        DelegatedAdministrator(
+                            arn=delegated_administrator["Arn"],
+                            id=delegated_administrator["Id"],
+                            name=delegated_administrator["Name"],
+                            email=delegated_administrator["Email"],
+                            status=delegated_administrator["Status"],
+                            joinedmethod=delegated_administrator["JoinedMethod"],
```

Comment:
Could you do the same here using dict.get()? Thanks!

Reply:
fixed.